README.md

```diff
@@ -1251,9 +1236,9 @@ paths:
                 type: object
 ```
 
-### Cross device URL - Premium Enterprise Feature
+### Cross device URL 
 
-This feature allows you to specify your own custom or whitelabel url that the cross device flow will redirect to instead of the Onfido default `id.onfido.com`. To use this feature generate a SDK token as shown below and use it to start the SDK.
+This premium enterprise feature allows you to specify your own custom or whitelabel url that the cross device flow will redirect to instead of the Onfido default `id.onfido.com`. To use this feature generate an SDK token as shown below and use it to start the SDK.
```

Comment:
Not sure if we should make the fact that it is a "premium enterprise feature" more prominent, which I think was why it was in the heading. @seewah thoughts?